server/src/main/java/io/littlehorse/common/model/getable/global/wfspec/TypeDefinitionModel.java

```diff
@@ -116,7 +117,8 @@ public void initFrom(Message proto, ExecutionContext ctx) {
                 this.structDefId = StructDefIdModel.fromProto(p.getStructDefId(), ctx);
                 break;
             case DEFINEDTYPE_NOT_SET:
-            default:
+                this.definedTypeCase = DefinedTypeCase.PRIMITIVE_TYPE;
+                this.primitiveType = VariableType.JSON_OBJ;
                 break;
         }
```

Comment:
This seems like it will work. It will mean that old servers are not compatible with new clients, but that is our contract.